src/Migrate/Changelog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Onboard Migrate to Private Link Common Cmdlets
```

Comment:
Migrate module does not change a thing, so change log should not be updated. Please remove this.

Reply:
Removed the changelog